schema/ClaimsAttributeObject.md

```diff
@@ -0,0 +1,36 @@
+# Attribute Object
+
+## Summary
+An Attribute Object contains attribute information the TDF system uses to enforce access control. Attributes are used in both the [PolicyObject](PolicyObject.md) to define the attributes that a subject "needs" to gain access in an ABAC sense, and in the [ClaimsObject](ClaimsObject.md) to assert the attributes that a subject "has" to satisfy the ABAC needs.
```

Comment:
Are we still using all of this?

Reply:
Currently yes - the OIDC work hasn't touched how policy currently works on the KAS end (though I think how we handle policy, especially clientside, needs a rework, and that's on our roadmap)